scripts/utils/update_package.py

```diff
@@ -205,6 +241,49 @@ def update_version_url(package):
     return latest_version
 
 
+def update_msixbundle_url(package):
+    install_script_path, content = get_install_script(package)
+
+    if not content:
+        return None
+
+    # Match urls like:
+    # - https://windbg.download.prss.microsoft.com/dbazure/prod/1-2502-25002-0/windbg.msixbundle
+    rex = re.search(
+        r"[\"'](https{0,1}:\/\/.+?[A-Za-z\-_]+\/(\d+\-\d+\-\d+\-\d+)\/[A-Za-z\-_]+\.msixbundle)[\"']", content
+    )
+
+    if not rex:
+        return None
+
+    sha256_rex = re.search(r"[\"']([A-Fa-f0-9]{64})[\"']", content)
+    if not sha256_rex:
+        return None
+
+    sha256 = sha256_rex.group(1)
+
+    latest_version = None
+
+    url = rex.group(1)
+    version = rex.group(2)
+    latest_version, latest_sha256 = get_msixbundle_version(url, version)
```

Comment:
[nitpick] As the variables are only used one, I would use them directly:

```suggestion
    latest_version, latest_sha256 = get_msixbundle_version(rex.group("url"), rex.group("version))
```